osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -262,6 +264,17 @@ protected Channel CurrentChannel
             {
                 if (currentChannel == value) return;
 
+                if (value.Id == CHANNEL_SELECTOR_ID)

```

Comment:
don't do this here. have ChatTabControl expose a bindable or action like `AddTabSelected`